composer.json

```diff
@@ -34,6 +34,7 @@
         "laminas/laminas-eventmanager": "^3.4",
         "laminas/laminas-servicemanager": "^3.21.0",
         "laminas/laminas-stdlib": "^3.10.1",
+        "laminas/laminas-translator": "^1.1",
```

Comment:
This swaps MVC (`-dev` dependency) for a non-dev dependency: is that really a good idea? 🤔

Reply:
Translator is a single interface pkg just for the translation contract. I think it's reasonable to import that symbol considering there's no equivalent PSR 🤷‍♂️

Reply:
Given this, I think we're fine merging :+1: